### PR TITLE
change

### PR DESCRIPTION
**Pull request by Josh-Diamond**
pull request :)

### CODE REVIEW

File: instagram/src/components/PostsContainer/PostContainer.js [hunk 1]
```diff
@@ -34,111 +78,55 @@ import SearchBar from '../SearchBar/SearchBar';
...
                     <img src={dummyData.imageUrl} alt={dummyData.username} />
-                 <div className='postText'>
-                        <div className='icons'>
-                            <i onClick={()=>this.increaseLikeHeandler(index)} class="far fa-heart fa-lg"></i>
-                            <i class="far fa-comment fa-lg"></i>
-                        </div>
-                    <p className='likes'>{dummyData.likes} likes</p>
+                 <PostText>
+                        <>
+                            <i onClick={()=>this.increaseLikeHandler(index)} className={dummyData.liked ? "fas fa-heart fa-lg" : "far fa-heart fa-lg"}></i>
```
Comment (gsamaniego41) [inline instagram/src/components/PostsContainer/PostContainer.js:114]: Nice job figuring out how to change the heart color :)

File: instagram/src/components/PostsContainer/PostContainer.js [hunk 2]
```diff
@@ -3,28 +3,72 @@ import CommentSection from '../CommentSection/CommentSection';
...
       componentDidMount() {
+            console.log('cdm', this.state.posts)
         this.setState({ posts: dummyData })
       }
       searchHandler = (e) => {
-          const filtered = this.state.posts.filter((post) => {
-            console.log(post)  
+          const filtered = this.state.posts.filter((post) => {  
+              console.log(post)
             return post.username.toLowerCase().includes(e.target.value.toLowerCase()) || post.likes.toString().includes(e.target.value.toString())
```
Comment (gsamaniego41) [inline instagram/src/components/PostsContainer/PostContainer.js:65]: Good job getting the search functionality to work and making it case insensitive

**Review by gsamaniego41 — COMMENTED** [review]
Nice job with MVP and stretches
✔️ Displays posts and associated comments
✔️ Filtered search
✔️ Increment/decrement a like
✔️ componentDidMount loads data to state
✔️ HOC login feature
✔️ Uses key prop on looped components
✔️ Adds comment
✔️ Uses localStorage
✔️ Styled components
✔️ Clean code
❌ I was not able to find PropTypes

**Comment by Josh-Diamond** [comment]
oops! i forgot proptypes! ..is this something i need to re-submit when
added? or just be aware of?

**Pull request merged**